src/main/groovy/jenkins/automation/builders/PipelineJobBuilder.groovy

```diff
@@ -0,0 +1,77 @@
+package jenkins.automation.builders
+
+import javaposse.jobdsl.dsl.DslFactory
+import javaposse.jobdsl.dsl.Job
+import jenkins.automation.utils.CommonUtils
+
+/**
+ * The basic PipelineJob builder
+ *
+ * @param stages list of maps describing the stages; see example linked below for the shape of the map
+ * @param name job name
+ * @param sandboxFlag, boolean to enable/disable sandbox, default true.
+ * @param description job description
+ * @param pipelineScript optional string containing the raw pipeline script (use instead of `stages`)
+ * @param pollScmSchedule optional string in cron format to trigger builds on a scheduled interval
+ * @see <a href="https://github.com/cfpb/jenkins-automation/blob/gh-pages/docs/examples.md#pipeline-builder" target="_blank">Pipeline job builder example</a>
+ */
+
+class PipelineJobBuilder {
+    List stages
+    String name
+    String description
+    String pipelineScript = ""
+    String pollScmSchedule
+    Boolean sandboxFlag = true
```

Comment:
My one gripe/nitpick is that sandbox here is set to `true`. Looking through your past commits it looks like it was `false` by default. Any reason for this change? I don't think I've come across stuff we're using that expects this to be sandboxed.

Reply:
It's a good point. I would have assumed `false` would be the default here. We don't want that?

Reply:
I take it all back -- I think we're good here 😅  http://unethicalblogger.com/2017/08/03/donut-disable-groovy-sandbox.html

Reply:
just to be clear, sandboxing by default seems good to me